packages/infrastructure/terraform.tfvars.template

```diff
@@ -0,0 +1,7 @@
+env = "test"
```

Comment:
Maybe just keeping this in the readme instead of in its own file? Similarly to what we did on [alv-website](https://github.com/Alv-no/alv-website/tree/master/packages/infrastructure).

Reply:
Done